### PR TITLE
recover features lost during newui develop merge

### PR DESCRIPTION
**Pull request by imgrby**
pan-drag bug fix.
2 view buttons.
userpage.
upload modals styles.
no harm to publish.

### CODE REVIEW

File: browser/scripts/ui.js [hunk 1]
```diff
@@ -171,7 +173,34 @@ VizorUI.prototype.onBtnAssetsClicked = function() {
 	this.syncVisibility();
 }
 
+VizorUI.prototype.enterVRView = function() {
+	var is_active = E2.app.worldEditor.isActive()
```
Comment (kschzt) [inline browser/scripts/ui.js]: camelCase

**Comment by kschzt** [comment]
lgtm otherwise

File: browser/scripts/ui.js (hunk 1 above)
Comment (imgrby) [inline browser/scripts/ui.js]: same as everywhere else, no?

Comment (gmarinov) [inline browser/scripts/ui.js]: do prefer camelCase for new code

**Comment by kschzt** [comment]
lgtm

**Pull request merged**